apps/app-article-server/README.md

```diff
@@ -8,18 +8,18 @@ Authentication is done via `AuthUser` and `Authorization` headers.
 
 ## Running the thing locally
 
-Simply execute
-
 ```
 export PUBLIC_URL=/dist
 yarn install
 yarn build
 yarn start
 ```
 
-As an alternative to exporting PUBLIC_URL for every new shell you can also use [direnv](https://direnv.net/docs/installation.html) to have it done automgically.
+As an alternative to exporting PUBLIC_URL for every new shell you can also use [direnv](https://direnv.net/docs/installation.html) to have it done automagically.
```

Comment:
Is there a reason why we can't set it by default?

Reply:
I don't think so. I've got used to setting it manually myself though.